src/Main.elm

```diff
@@ -37,20 +37,67 @@ update msg model =
 
 ---- VIEW ----
 
-
 view : Model -> Html Msg
 view model =
-    Grid.container []
+    container []
             [ CDN.stylesheet
-            , Grid.gridx [] []
-            , p [] []
-            ]
-
+            , CDN.stylesheetFont
+            , CDN.stylesheetTheme 
+            , h1 [] 
+                [ text "Elm Foundation"
+                  , h4 [] [text "XY Grid"]
+                ]
+            , grid [gridX, gridMarginx, gridPaddingy, gridMarginy]
```

Comment:
Some ideas
```elm
html =
    grid [ container Full, direction X, gutter MarginX ]
        [ cell [ size Small4 ] [ text "cell" ]
        , cell [ size Small4 ] [ text "cell" ]
        , cell [ size Auto ] [ text "cell" ]
        ]
```

Reply:
This looks really neat. I'll try this out.